### PR TITLE
Changing to new storage account

### PR DESCRIPTION
**Pull request by fcabrera23**

### CODE REVIEW

File: azure_arc_data_jumpstart/aks/ARM/artifacts/DeployPostgreSQL.ps1 [hunk 1]
```diff
@@ -74,7 +74,7 @@ Start-Sleep -Seconds 60
 # Downloading demo database and restoring onto Postgres
 Write-Host "`n"
 Write-Host "Downloading AdventureWorks.sql template for PostgreSQL (1/3)"
-kubectl exec $pgWorkerPodName -n arc -c postgres -- /bin/bash -c "curl -o /tmp/AdventureWorks2019.sql 'https://jsvhds.blob.core.windows.net/sampledb/AdventureWorks2019.sql?sp=r&st=2023-08-10T18:30:33Z&se=2033-08-11T02:30:33Z&spr=https&sv=2022-11-02&sr=b&sig=5kD1PR4gwaCWlefSknggq%2BYsx1FgBrp5Kv1pH42d1nE%3D'" 2>&1 | Out-Null
+kubectl exec $pgWorkerPodName -n arc -c postgres -- /bin/bash -c "curl -o /tmp/AdventureWorks2019.sql 'https://jumpstartprodsg.blob.core.windows.net/sampledb/AdventureWorks2019.sql?sp=r&st=2023-08-10T18:30:33Z&se=2033-08-11T02:30:33Z&spr=https&sv=2022-11-02&sr=b&sig=5kD1PR4gwaCWlefSknggq%2BYsx1FgBrp5Kv1pH42d1nE%3D'" 2>&1 | Out-Null
```
Comment (dkirby-ms) [inline azure_arc_data_jumpstart/aks/ARM/artifacts/DeployPostgreSQL.ps1:77]: We still use a SAS? I thought this was not allowed?

File: azure_arc_data_jumpstart/aro/ARM/artifacts/DeployPostgreSQL.ps1 [hunk 2]
```diff
@@ -74,7 +74,7 @@ Start-Sleep -Seconds 60
 # Downloading demo database and restoring onto Postgres
 Write-Host "`n"
 Write-Host "Downloading AdventureWorks.sql template for PostgreSQL (1/3)"
-kubectl exec $pgWorkerPodName -n arc -c postgres -- /bin/bash -c "curl -o /tmp/AdventureWorks2019.sql 'https://jsvhds.blob.core.windows.net/sampledb/AdventureWorks2019.sql?sp=r&st=2023-08-10T18:30:33Z&se=2033-08-11T02:30:33Z&spr=https&sv=2022-11-02&sr=b&sig=5kD1PR4gwaCWlefSknggq%2BYsx1FgBrp5Kv1pH42d1nE%3D'" 2>&1 | Out-Null
+kubectl exec $pgWorkerPodName -n arc -c postgres -- /bin/bash -c "curl -o /tmp/AdventureWorks2019.sql 'https://jumpstartprodsg.blob.core.windows.net/sampledb/AdventureWorks2019.sql?sp=r&st=2023-08-10T18:30:33Z&se=2033-08-11T02:30:33Z&spr=https&sv=2022-11-02&sr=b&sig=5kD1PR4gwaCWlefSknggq%2BYsx1FgBrp5Kv1pH42d1nE%3D'" 2>&1 | Out-Null
```
Comment (dkirby-ms) [inline azure_arc_data_jumpstart/aro/ARM/artifacts/DeployPostgreSQL.ps1:77]: SAS token present

File: azure_arc_data_jumpstart/cluster_api/capi_azure/ARM/artifacts/DeployPostgreSQL.ps1 [hunk 3]
```diff
@@ -74,7 +74,7 @@ Start-Sleep -Seconds 60
 # Downloading demo database and restoring onto Postgres
 Write-Host "`n"
 Write-Host "Downloading AdventureWorks.sql template for PostgreSQL (1/3)"
-kubectl exec $pgWorkerPodName -n arc -c postgres -- /bin/bash -c "curl -o /tmp/AdventureWorks2019.sql 'https://jsvhds.blob.core.windows.net/sampledb/AdventureWorks2019.sql?sp=r&st=2023-08-10T18:30:33Z&se=2033-08-11T02:30:33Z&spr=https&sv=2022-11-02&sr=b&sig=5kD1PR4gwaCWlefSknggq%2BYsx1FgBrp5Kv1pH42d1nE%3D'" 2>&1 | Out-Null
+kubectl exec $pgWorkerPodName -n arc -c postgres -- /bin/bash -c "curl -o /tmp/AdventureWorks2019.sql 'https://jumpstartprodsg.blob.core.windows.net/sampledb/AdventureWorks2019.sql?sp=r&st=2023-08-10T18:30:33Z&se=2033-08-11T02:30:33Z&spr=https&sv=2022-11-02&sr=b&sig=5kD1PR4gwaCWlefSknggq%2BYsx1FgBrp5Kv1pH42d1nE%3D'" 2>&1 | Out-Null
```
Comment (dkirby-ms) [inline azure_arc_data_jumpstart/cluster_api/capi_azure/ARM/artifacts/DeployPostgreSQL.ps1:77]: SAS token

File: azure_arc_data_jumpstart/eks/terraform/artifacts/DeployPostgreSQL.ps1 [hunk 4]
```diff
@@ -74,7 +74,7 @@ Start-Sleep -Seconds 60
 # Downloading demo database and restoring onto Postgres
 Write-Host "`n"
 Write-Host "Downloading AdventureWorks.sql template for PostgreSQL (1/3)"
-kubectl exec $pgWorkerPodName -n arc -c postgres -- /bin/bash -c "curl -o /tmp/AdventureWorks2019.sql 'https://jsvhds.blob.core.windows.net/sampledb/AdventureWorks2019.sql?sp=r&st=2023-08-10T18:30:33Z&se=2033-08-11T02:30:33Z&spr=https&sv=2022-11-02&sr=b&sig=5kD1PR4gwaCWlefSknggq%2BYsx1FgBrp5Kv1pH42d1nE%3D'" 2>&1 | Out-Null
+kubectl exec $pgWorkerPodName -n arc -c postgres -- /bin/bash -c "curl -o /tmp/AdventureWorks2019.sql 'https://jumpstartprodsg.blob.core.windows.net/sampledb/AdventureWorks2019.sql?sp=r&st=2023-08-10T18:30:33Z&se=2033-08-11T02:30:33Z&spr=https&sv=2022-11-02&sr=b&sig=5kD1PR4gwaCWlefSknggq%2BYsx1FgBrp5Kv1pH42d1nE%3D'" 2>&1 | Out-Null
```
Comment (dkirby-ms) [inline azure_arc_data_jumpstart/eks/terraform/artifacts/DeployPostgreSQL.ps1:77]: SAS token

File: azure_arc_data_jumpstart/gke/terraform/artifacts/DeployPostgreSQL.ps1 [hunk 5]
```diff
@@ -73,7 +73,7 @@ Start-Sleep -Seconds 60
 
 # Downloading demo database and restoring onto Postgres
 Write-Host "Downloading AdventureWorks.sql template for PostgreSQL (1/3)"
-kubectl exec $pgCoordinatorPodName -n arc -c postgres -- /bin/bash -c "curl -o /tmp/AdventureWorks2019.sql 'https://jsvhds.blob.core.windows.net/sampledb/AdventureWorks2019.sql?sp=r&st=2023-08-10T18:30:33Z&se=2033-08-11T02:30:33Z&spr=https&sv=2022-11-02&sr=b&sig=5kD1PR4gwaCWlefSknggq%2BYsx1FgBrp5Kv1pH42d1nE%3D'" 2>&1 | Out-Null
+kubectl exec $pgCoordinatorPodName -n arc -c postgres -- /bin/bash -c "curl -o /tmp/AdventureWorks2019.sql 'https://jumpstartprodsg.blob.core.windows.net/sampledb/AdventureWorks2019.sql?sp=r&st=2023-08-10T18:30:33Z&se=2033-08-11T02:30:33Z&spr=https&sv=2022-11-02&sr=b&sig=5kD1PR4gwaCWlefSknggq%2BYsx1FgBrp5Kv1pH42d1nE%3D'" 2>&1 | Out-Null
```
Comment (dkirby-ms) [inline azure_arc_data_jumpstart/gke/terraform/artifacts/DeployPostgreSQL.ps1:76]: SAS token

File: azure_arc_data_jumpstart/kubeadm/azure/ARM/artifacts/DeployPostgreSQL.ps1 [hunk 6]
```diff
@@ -75,7 +75,7 @@ Start-Sleep -Seconds 60
 # Downloading demo database and restoring onto Postgres
 Write-Host "`n"
 Write-Host "Downloading AdventureWorks.sql template for PostgreSQL (1/3)"
-kubectl exec $pgWorkerPodName -n arc -c postgres -- /bin/bash -c "curl -o /tmp/AdventureWorks2019.sql 'https://jsvhds.blob.core.windows.net/sampledb/AdventureWorks2019.sql?sp=r&st=2023-08-10T18:30:33Z&se=2033-08-11T02:30:33Z&spr=https&sv=2022-11-02&sr=b&sig=5kD1PR4gwaCWlefSknggq%2BYsx1FgBrp5Kv1pH42d1nE%3D'" 2>&1 | Out-Null
+kubectl exec $pgWorkerPodName -n arc -c postgres -- /bin/bash -c "curl -o /tmp/AdventureWorks2019.sql 'https://jumpstartprodsg.blob.core.windows.net/sampledb/AdventureWorks2019.sql?sp=r&st=2023-08-10T18:30:33Z&se=2033-08-11T02:30:33Z&spr=https&sv=2022-11-02&sr=b&sig=5kD1PR4gwaCWlefSknggq%2BYsx1FgBrp5Kv1pH42d1nE%3D'" 2>&1 | Out-Null
```
Comment (dkirby-ms) [inline azure_arc_data_jumpstart/kubeadm/azure/ARM/artifacts/DeployPostgreSQL.ps1:78]: SAS token

File: azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_full/bicep_template/artifacts/LogonScript.ps1 [hunk 7]
```diff
@@ -14,7 +14,7 @@ $azureLocation = $env:azureLocation
 $spnClientId = $env:spnClientId
 $spnClientSecret = $env:spnClientSecret
 $spnTenantId = $env:spnTenantId
-$vhdxUri = "https://jsvhds.blob.core.windows.net/scenarios/prod/JSW11IoTBase.vhdx?sp=r&st=2023-05-09T12:36:32Z&se=2033-05-09T20:36:32Z&spr=https&sv=2022-11-02&sr=b&sig=xFROrqGkKDIdrXqiAMLZGLEwTMToOWoNNDMVz1zvPMc%3D"
+$vhdxUri = "https://jumpstartprodsg.blob.core.windows.net/scenarios/prod/JSW11IoTBase.vhdx?sp=r&st=2023-05-09T12:36:32Z&se=2033-05-09T20:36:32Z&spr=https&sv=2022-11-02&sr=b&sig=xFROrqGkKDIdrXqiAMLZGLEwTMToOWoNNDMVz1zvPMc%3D"
```
Comment (dkirby-ms) [inline azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_full/bicep_template/artifacts/LogonScript.ps1:17]: SAS

File: azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_full_akri/bicep_template/artifacts/LogonScript.ps1 [hunk 8]
```diff
@@ -14,7 +14,7 @@ $azureLocation = $env:azureLocation
 $spnClientId = $env:spnClientId
 $spnClientSecret = $env:spnClientSecret
 $spnTenantId = $env:spnTenantId
-$vhdxUri = "https://jsvhds.blob.core.windows.net/scenarios/prod/JSW11IoTBase.vhdx?sp=r&st=2023-05-09T12:36:32Z&se=2033-05-09T20:36:32Z&spr=https&sv=2022-11-02&sr=b&sig=xFROrqGkKDIdrXqiAMLZGLEwTMToOWoNNDMVz1zvPMc%3D"
+$vhdxUri = "https://jumpstartprodsg.blob.core.windows.net/scenarios/prod/JSW11IoTBase.vhdx?sp=r&st=2023-05-09T12:36:32Z&se=2033-05-09T20:36:32Z&spr=https&sv=2022-11-02&sr=b&sig=xFROrqGkKDIdrXqiAMLZGLEwTMToOWoNNDMVz1zvPMc%3D"
```
Comment (dkirby-ms) [inline azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_full_akri/bicep_template/artifacts/LogonScript.ps1:17]: SAS

File: azure_arc_servers_jumpstart/esu/artifacts/LogonScript.ps1 [hunk 9]
```diff
@@ -139,7 +139,7 @@ Start-Transcript -Path $logFilePath -Force -ErrorAction SilentlyContinue
         $imageName = "JSWin2K12Base"
         $vmvhdPath = "$Env:ESUVMDir\${imageName}.vhdx"
         # Moved VHD storage account details here to keep only in place to prevent duplicates.
-        $vhdDownload = "https://jsvhds.blob.core.windows.net/scenarios/prod/JSWin2K12Base.vhdx?sp=r&st=2023-09-11T08:05:53Z&se=2025-11-08T17:05:53Z&spr=https&sv=2022-11-02&sr=b&sig=zoVpd9AMzsTRRE0a7eLJYeFURexY4R9VSzOGKfLOx%2FQ%3D"
+        $vhdDownload = "https://jumpstartprodsg.blob.core.windows.net/scenarios/prod/JSWin2K12Base.vhdx?sp=r&st=2023-09-11T08:05:53Z&se=2025-11-08T17:05:53Z&spr=https&sv=2022-11-02&sr=b&sig=zoVpd9AMzsTRRE0a7eLJYeFURexY4R9VSzOGKfLOx%2FQ%3D"
```
Comment (dkirby-ms) [inline azure_arc_servers_jumpstart/esu/artifacts/LogonScript.ps1:142]: SAS

File: azure_arc_servers_jumpstart/esu/artifacts/LogonScript.ps1 [hunk 10]
```diff
@@ -209,7 +209,7 @@ Start-Transcript -Path $logFilePath -Force -ErrorAction SilentlyContinue
         $imageName = "JSSQL12Base"
         $vmvhdPath = "$Env:ESUVMDir\${imageName}.vhdx"
         # Moved VHD storage account details here to keep only in place to prevent duplicates.
-        $vhdDownload = "https://jsvhds.blob.core.windows.net/scenarios/prod/JSSQL12Base.vhdx?sp=r&st=2023-09-27T06:57:38Z&se=2027-09-11T14:57:38Z&spr=https&sv=2022-11-02&sr=b&sig=BXtEL%2B7RdLairRHXd3TA6n5q%2FNktjItvcU1rzol9Dl0%3D"
+        $vhdDownload = "https://jumpstartprodsg.blob.core.windows.net/scenarios/prod/JSSQL12Base.vhdx?sp=r&st=2023-09-27T06:57:38Z&se=2027-09-11T14:57:38Z&spr=https&sv=2022-11-02&sr=b&sig=BXtEL%2B7RdLairRHXd3TA6n5q%2FNktjItvcU1rzol9Dl0%3D"
```
Comment (dkirby-ms) [inline azure_arc_servers_jumpstart/esu/artifacts/LogonScript.ps1:212]: SAS

File: azure_arc_servers_jumpstart/esu/artifacts/LogonScript.ps1 [hunk 11]
```diff
@@ -278,15 +278,15 @@ Start-Transcript -Path $logFilePath -Force -ErrorAction SilentlyContinue
             JSWin2K12Base = @{
                 imageName = 'JSWin2K12Base'
                 vmvhdPath = "$Env:ESUVMDir\JSWin2K12Base.vhdx"
-                vhdDownload = "https://jsvhds.blob.core.windows.net/scenarios/prod/JSWin2K12Base.vhdx?sp=r&st=2023-09-11T08:05:53Z&se=2025-11-08T17:05:53Z&spr=https&sv=2022-11-02&sr=b&sig=zoVpd9AMzsTRRE0a7eLJYeFURexY4R9VSzOGKfLOx%2FQ%3D"
+                vhdDownload = "https://jumpstartprodsg.blob.core.windows.net/scenarios/prod/JSWin2K12Base.vhdx?sp=r&st=2023-09-11T08:05:53Z&se=2025-11-08T17:05:53Z&spr=https&sv=2022-11-02&sr=b&sig=zoVpd9AMzsTRRE0a7eLJYeFURexY4R9VSzOGKfLOx%2FQ%3D"
```
Comment (dkirby-ms) [inline azure_arc_servers_jumpstart/esu/artifacts/LogonScript.ps1:281]: SAS

File: azure_jumpstart_arcbox/artifacts/DeployPostgreSQL.ps1 [hunk 12]
```diff
@@ -69,7 +69,7 @@ Start-Sleep -Seconds 60
 
 # Downloading demo database and restoring onto Postgres
 Write-Host "Downloading AdventureWorks.sql template for Postgres... (1/3)"
-kubectl exec $pgWorkerPodName -n arc -c postgres -- /bin/bash -c "curl -o /tmp/AdventureWorks2019.sql 'https://jsvhds.blob.core.windows.net/sampledb/AdventureWorks2019.sql?sp=r&st=2023-08-10T18:30:33Z&se=2033-08-11T02:30:33Z&spr=https&sv=2022-11-02&sr=b&sig=5kD1PR4gwaCWlefSknggq%2BYsx1FgBrp5Kv1pH42d1nE%3D'" 2>&1 | Out-Null
+kubectl exec $pgWorkerPodName -n arc -c postgres -- /bin/bash -c "curl -o /tmp/AdventureWorks2019.sql 'https://jumpstartprodsg.blob.core.windows.net/sampledb/AdventureWorks2019.sql?sp=r&st=2023-08-10T18:30:33Z&se=2033-08-11T02:30:33Z&spr=https&sv=2022-11-02&sr=b&sig=5kD1PR4gwaCWlefSknggq%2BYsx1FgBrp5Kv1pH42d1nE%3D'" 2>&1 | Out-Null
```
Comment (dkirby-ms) [inline azure_jumpstart_arcbox/artifacts/DeployPostgreSQL.ps1:72]: SAS

**Review by dkirby-ms — COMMENTED** [review]
SAS tokens present

File: azure_arc_data_jumpstart/aks/ARM/artifacts/DeployPostgreSQL.ps1 (hunk 1 above)
Comment (fcabrera23) [reply azure_arc_data_jumpstart/aks/ARM/artifacts/DeployPostgreSQL.ps1]: Fixed

**Comment by fcabrera23** [comment]
@dkirby-ms  thanks for reviewing - I've removed the SAS tokens also as it's no longer required.

**Review by sebassem — APPROVED** [review]

**Review by dkirby-ms — APPROVED** [review]

**Pull request merged**